client/src/util/constants.ts

```diff
@@ -7,3 +11,87 @@ export const MESSAGE = {
   NO_RESULT_FOUND: "No result found. It might be because:",
   UNSCHEDULED_COURSE_WARNING: "Warning: Your timetable contains one or more courses that are not shown below",
 };
+export const team = [
+  {
+    avatarURL: nico,
+    name: "Nico Zhu",
+    role: "Designer"
+  },
+  {
+    avatarURL: sarah,
+    name: "Sarah Dang",
```

Comment:
Should we be consistent with either only first name, or both first name last name?

Reply:
Good point!